packages/nimble-components/src/label-provider/table/index.ts

```diff
@@ -0,0 +1,96 @@
+import { attr } from '@microsoft/fast-element';
+import { DesignSystem } from '@microsoft/fast-foundation';
+import { LabelProviderBase } from '../base';
+import {
+    tableCellActionMenuLabel,
+    tableColumnHeaderGroupedIndicatorLabel,
+    tableGroupCollapseLabel,
+    tableGroupExpandLabel,
+    tableGroupsCollapseAllLabel
+} from './label-tokens';
+import { tableLabelDefaults } from './label-token-defaults';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-label-provider-table': LabelProviderTable;
+    }
+}
+
+/**
+ * Label provider for the Nimble table (and its sub-components and columns)
+ */
+export class LabelProviderTable extends LabelProviderBase {
```

Comment:
Note: The [HLD](https://github.com/ni/nimble/tree/main/specs/labels-and-localization#plan-for-nimble-provided-labels) had a few labels whose tokens are not being added here yet - namely those that will be used for the column header menu & menu items, since it's not implemented yet. I figured we can add the tokens once we work on that (but if anyone wants to argue for having the tokens now, in the name of fewer label changes for clients, let me know)

Reply:
I'm alright adding them when we need them